pandas/core/generic.py

```diff
@@ -10460,7 +10460,8 @@ def _add_series_or_dataframe_operations(cls):
         Add the series or dataframe only operations to the cls; evaluate
         the doc strings again.
         """
-        from pandas.core.window import EWM, Expanding, Rolling, Window
+        from pandas.core.window import ExponentialMovingWindow, Expanding, Rolling
+        from pandas.core.window import Window
```

Comment:
```suggestion
        from pandas.core.window import Expanding, ExponentialMovingWindow, Rolling, Window
```

Reply:
if it is too long (>88), you might need to use `black` for reformatting

Reply:
found a workaround by using an absolute import would that be ok?

Reply:
```python
from pandas.core.window import (
     Expanding,
     ExponentialMovingWindow,
     Rolling,
     Window,
)
```

I just tried black, and it looks like above.

just a general tip, we use black and isort as formatting tools, described in https://pandas.pydata.org/pandas-docs/stable/development/contributing.html#python-pep8-black , you could see that, after code change, it's suggested to run black and isort to make sure formatting is standardized and then able to pass the CI.

It is probably functionally okay to have this workaround, but preferably use absolute import as in all files in pandas.

Reply:
ok np

Reply:
aah, that's why the Cl test was failing I see.

Reply:
k np I've changed it to use black formatting instead of the absolute import.

Reply:
I used the absolute import because that's what pycharm suggested for me.